[2022-12-11 14:35:35,654] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T14:35:33.520021+00:00 [queued]>
[2022-12-11 14:35:35,668] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T14:35:33.520021+00:00 [queued]>
[2022-12-11 14:35:35,669] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 14:35:35,670] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 14:35:35,671] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 14:35:35,693] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-12-11 14:35:33.520021+00:00
[2022-12-11 14:35:35,698] {standard_task_runner.py:52} INFO - Started process 954 to run task
[2022-12-11 14:35:35,703] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'manual__2022-12-11T14:35:33.520021+00:00', '--job-id', '513', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpkjied7go', '--error-file', '/tmp/tmpdm7zsclu']
[2022-12-11 14:35:35,706] {standard_task_runner.py:80} INFO - Job 513: Subtask world
[2022-12-11 14:35:35,791] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world manual__2022-12-11T14:35:33.520021+00:00 [running]> on host 7e39590a7807
[2022-12-11 14:35:35,895] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T14:35:33.520021+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T14:35:33.520021+00:00
[2022-12-11 14:35:35,898] {logging_mixin.py:115} INFO - Airflow
[2022-12-11 14:35:35,900] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 14:35:35,923] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20221211T143533, start_date=20221211T143535, end_date=20221211T143535
[2022-12-11 14:35:35,973] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 14:35:36,035] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
